tools/plist_to_html/setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="plist-to-html",
+    version="1.0.0",
```

Comment:
I think we should start the versions with `0.1.0` or `0.0.1`. `1.0` usually means a production ready stable release.